roles/grafana_agent/tasks/main.yml

```diff
@@ -32,6 +32,23 @@
   when: ansible_pkg_mgr == "apt"
   register: keyrings_exists
 
+- name: Remove conflicting Grafana key file if exist
+  become: true
```

Comment:
This is not necessary.  The entire role is `become: true`.  https://github.com/ceph/ceph-cm-ansible/blob/main/grafana_agent.yml#L6